Cargo.toml

```diff
@@ -69,8 +70,9 @@ solana-rayon-threadlimit = "=1.17.28"
 solana-runtime = "=1.17.28"
 solana-sdk = "=1.17.28"
 solana-streamer = "=1.17.28"
+solana-validator = "=1.17.28"
 thiserror = "1.0.40"
-tikv-jemallocator = { version = "0.5", features = ["profiling"] }
+tikv-jemallocator = { version = "0.4", features = ["profiling"] }
```

Comment:
Why was the version number of this allocator crate reduced from 0.5 to 0.4?

Reply:
dependency hell

`error: failed to select a version for `tikv-jemalloc-sys`.
    ... required by package `tikv-jemallocator v0.5.0`
    ... which satisfies dependency `tikv-jemallocator = "^0.5"` of package `jito-transaction-relayer v0.1.12 (/home/jed/jito/jito-relayer/transaction-relayer)`
versions that meet the requirements `^0.5.0` are: 0.5.4+5.3.0-patched, 0.5.3+5.3.0-patched, 0.5.2+5.3.0-patched, 0.5.1+5.3.0-patched, 0.5.0+5.3.0

the package `tikv-jemalloc-sys` links to the native library `jemalloc`, but it conflicts with a previous package which links to `jemalloc` as well:
package `tikv-jemalloc-sys v0.4.0+5.2.1`
    ... which satisfies dependency `tikv-jemalloc-sys = "^0.4.0"` of package `tikv-jemallocator v0.4.1`
    ... which satisfies dependency `jemallocator = "^0.4.1"` of package `solana-validator v1.17.31`
    ... which satisfies dependency `solana-validator = "=1.17.31"` of package `jito-transaction-relayer v0.1.12 (/home/jed/jito/jito-relayer/transaction-relayer)`
Only one package in the dependency graph may specify the same links value. This helps ensure that only one copy of a native library is linked in the final binary. Try to adjust your dependencies so that only one package uses the `links = "jemalloc"` value. For more information, see https://doc.rust-lang.org/cargo/reference/resolver.html#links.

failed to select a version for `tikv-jemalloc-sys` which could resolve this conflict`